src/main/java/meditracker/command/DeleteCommand.java

```diff
@@ -24,7 +25,7 @@ public class DeleteCommand extends Command {
      * Constructs a DeleteCommand object with the specified arguments.
      * @param arguments The arguments containing medication information to be parsed.
      */
```

Comment:
Missing JavaDoc for DuplicateArgumentFoundException.

Reply:
Resolved in 236b662